### PR TITLE
cleanup constants for scheduler metrics

### PR DESCRIPTION
**Pull request by 249043822**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by alculquicondor** [comment]
/ok-to-test

Please fix the release note.

**Review by damemi — COMMENTED** [review]
/release-note-none

**Comment by damemi** [comment]
@249043822 for future reference, non user-facing changes the release note block needs to look like this (or else the bot won't pick it up):
Screen Shot 2020-12-11 at 11 00 49 AM

**Comment by 249043822** [comment]
@damemi ok， thanks

File: pkg/scheduler/metrics/metrics.go [hunk 1]
```diff
@@ -28,13 +28,10 @@ import (
 const (
 	// SchedulerSubsystem - subsystem name used by scheduler
 	SchedulerSubsystem = "scheduler"
-
-	// OperationLabel - operation label name
-	OperationLabel = "operation"
 	// Below are possible values for the operation label. Each represents a substep of e2e scheduling:
 
-	// PreemptionEvaluation - preemption evaluation operation label value (occurs in case of scheduling fitError).
-	PreemptionEvaluation = "preemption_evaluation"
```
Comment (oomichi) [inline pkg/scheduler/metrics/metrics.go:37]: This has been not used since c62f0dd165af2e4bbc1dd0521b89edc70236e9fa

**Review by oomichi — COMMENTED** [review]
/lgtm

**Pull request merged**